docs/ref/models/database-functions.txt

```diff
@@ -1982,3 +1982,63 @@ gaps.
 Computes the row number according to the ordering of either the frame clause
 or the ordering of the whole query if there is no partitioning of the
 :ref:`window frame <window-frames>`.
+
+.. json-related-functions:
+
+JSON-related functions
+======================
+
+We'll be using the following model in examples of each function::
+
+    class UserPreferences(models.Model):
+        settings = models.JSONField(encoder=DjangoJSONEncoder)
+
+``JSONSet``
+-----------
+
```

Comment:
```suggestion

.. versionadded:: 5.2

```

I also think it might be worth adding a release not for `JSONSet` in the "Minor features" section as part of this commit (3600f05c99bdb767b17da1715175ef184ced8e75)